src/server.js

```diff
@@ -116,11 +120,29 @@ const main = async () => {
       event: eventSchema,
       properties: propertiesSchema.tailor(req.user ? 'authenticated' : 'unauthenticated')
     }))
+
     const data = _.update('properties', properties => ({
       ...properties,
       token,
       'distinct_id': req.user ? userDistinctId(req.user) : properties.distinct_id
     }), req.body)
+
+    /**
+     * Temporary code to ignore the 'request:failed' event and delay the repsonse to
+     * slow the browser requests down and reduce the errors, and getting the client out of
+     * the degenerative state
```

Comment:
might be good to add to this comment any links to ticket or discussion on the noted "degenerative state" for future context